docs/reference/system/permissions.md

```diff
@@ -349,7 +349,7 @@ POST /graphql/system
 
 ```graphql
 type Mutation {
-	create_permissions_itemss(data: [create_directus_permissions_input!]!): [directus_permissions]
+	create_permissions_items(data: [create_directus_permissions_input!]!): [directus_permissions]
```

Comment:
scary

Reply:
Should be fine. This is the docs example, which has a typo. Doesn't change anything in the actual code 👍🏻